lib/runner.js

```diff
@@ -63,12 +63,11 @@ function setupRemoteSessionAndRunTests(config, options) {
             browserName: options.browser,
             platform: "ANY"
         };
-    browserOptions['browserstack.local'] = options.browserstackLocal;
     let bsOptions = {
         "logLevel": options.verbose ? "verbose" : "silent",
         "desiredCapabilities": browserOptions,
-        "user": process.env.BROWSERSTACK_USER,
-        "key": process.env.BROWSERSTACK_KEY,
+        "user": config.browserstack ? config.browserstack.user : '',
```

Comment:
How about the below, as it isn't directly related to browserstack. Any remote selenium server can have a username and password associated with it.

"user": config.user,
"key": config.key